conf/digestwriter.env

```diff
@@ -3,6 +3,7 @@ POSTGRES_PASSWORD=archive_db_writer_pwd
 LOGGING_LEVEL=DEBUG
 KAFKA_BROKER_CONSUMER_GROUP=consumer_group
 KAFKA_BROKER_ADDRESS=kafka:9092
+KAFKA_SERVERS=broker1.example.com:9092,broker2.example.com:9092,broker3.example.com:9092
```

Comment:
This should not contain example values but real values for local deployment - kafka:9092 as previously.